app/src/main/java/me/hufman/androidautoidrive/carapp/music/views/BrowsePageView.kt

```diff
@@ -28,7 +28,7 @@ enum class BrowseAction(val getLabel: () -> String) {
 		return getLabel()
 	}
 }
-class BrowsePageView(val state: RHMIState, val musicImageIDs: MusicImageIDs, val browsePageModel: BrowsePageModel, val browseController: BrowsePageController, var previouslySelected: MusicMetadata?, val graphicsHelpers: GraphicsHelpers): CoroutineScope {
+class BrowsePageView(val state: RHMIState, musicImageIDs: MusicImageIDs, val browsePageModel: BrowsePageModel, val browseController: BrowsePageController, var previouslySelected: MusicMetadata?, val graphicsHelpers: GraphicsHelpers): CoroutineScope {
```

Comment:
I'm preparing a refactor of the BrowsePageView on the [browsepage_model](https://github.com/hufman/AndroidAutoIdrive/compare/browsepage_model) branch, which simplifies this class a lot and remove any special-casing for this extra Search Results usecase. After I finish that, this module should be much more straight-forward.